ui/oopsyraidsy/oopsyraidsy.js

```diff
@@ -917,7 +917,7 @@ class DamageTracker {
   OnTrigger(trigger, evt, matches) {
     // If using named groups, treat matches.groups as matches
     // so triggers can do things like matches.target.
-    if ((matches != undefined) && (matches.groups != undefined))
+    if ((matches !== undefined) && (matches.groups !== undefined))
```

Comment:
I looked through callsites, and I think matches is always defined, so I think this is fine?  (And one callsite already passes matches.groups through instead of just matches, which I guess works out in the end, but is a bit asymmetrical.)